gateway/conf/nginx.conf.liquid

```diff
@@ -61,37 +67,38 @@ http {
     {% include file %}
   {% endfor %}
 
+
+  {% if jaeger_enabled %}
+    {% include "conf.d/jaeger.conf" %}
```

Comment:
We can have a `opentracing` subfolder there and do: 

```liquid
{% include "conf.d/opentracing" | append opentracing_tracer | append ".conf" %}
```

So we have a conf file for each supported tracer.